core/src/test/java/com/opensymphony/xwork2/ModelDrivenAction.java

```diff
@@ -44,7 +44,6 @@ public String getFoo() {
     /**
      * @return the model to be pushed onto the ValueStack after the Action itself
      */
-    @StrutsParameter(depth = 2)
```

Comment:
I removed all these annotations on `ModelDriven` Action model getters as they are no longer required (not that they were being detected correctly previously)